CQUI.modinfo

```diff
@@ -626,7 +629,7 @@
     </ImportFiles>
 
     <!-- ============================== UNITREPORT ============================== -->
-    <! -- UnitReport was commented out prior to May 2020 patch -->
+    <!-- UnitReport was commented out prior to May 2020 patch -->
```

Comment:
thank you for fixing this. I was looking for a way to report a bug in that Uploader, but that is not trivial. there is no link to developer's page , no support info, no docs, nothing. 
the uploader should not crash if there is a syntax error in the file it tries to upload :)